Pokedex/MainScreen.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ViewController.swift
+//  Pokedex
+//
+//  Created by Стефан  Миливоевич on 5.10.22..
+//
+
+import UIKit
+
+class MainScreen: UIViewController, UITableViewDelegate, UITableViewDataSource  {
```

Comment:
Удели время naming. Почитай статейки, которые я тебе скинул. В данном случае это не совсем главный экран - скорее это главный VC, а в контексте приложения это контроллер списка покемонов, кажется, лаконичным было бы название "PokemonListViewController" или что-то вроде этого.
```suggestion
class PokemonListViewController: UIViewController, UITableViewDelegate, UITableViewDataSource  {
```
Ведь будет ещё и детальный экран, например, PokemonDetailsViewController.
Чуть больше смысла в названиях